player/video.c

```diff
@@ -343,7 +343,7 @@ static void adjust_sync(struct MPContext *mpctx, double v_pts, double frame_time
 {
     struct MPOpts *opts = mpctx->opts;
 
-    if (mpctx->audio_status != STATUS_PLAYING)
+    if (mpctx->audio_status == STATUS_EOF)
```

Comment:
This breaks that one sample because it was stuck in `STATUS_READY` forever. We can add make this check `STATUS_PLAYING` and `STATUS_DRAINING` if that helps.

Reply:
We only call adjust_sync for PLAYING/DRAINING/EOF

https://github.com/mpv-player/mpv/blob/ff209a831ebb8667e40d50f63dea9136a1473d40/player/video.c#L391

Reply:
That's for video. Here it's the audio status that's the problem.

Reply:
Hmm, I'm not convinced anymore this is the right fix. Checking for only playing/draining also breaks that sample.